app/models/user.rb

```diff
@@ -111,6 +111,7 @@ def build_from_invitation(invitation)
                          name: invitation.name,
                          role: invitation.role,
                          phantom: invitation.phantom,
+                         timeline_algorithm: invitation.timeline_algorithm,
```

Comment:
I think the timeline_algorithm can fall back to invitation.course.default_timeline_algorithm. If the invitation does not specify any timeline_algorithm, we will then default to the course's default timeline. What do you think?

Reply:
Yup, good idea. I didn't include it here since I assumed `set_default` would be able to override this, but it doesn't seem to be the case. I will add a fallback here.